about.html

```diff
@@ -68,3 +68,39 @@ <h1 class="openlex-header">OpenLexington. Open Data. Open Government.</h1>
       <li>Pursue our mission through other actions that do not endanger the status of OpenLexington</li>
     </ul>
   </div>
+</div>
+<hr>
+
+<div class="container">
+  <h2>Who We Are</h2>
+
+  <div class="col-xs-12 col-sm-6 col-lg-4">
+    <h3>Core Members</h3>
+    <ul>
+      <li><b>Captain(s)</b><br /> Chase Southard, Mike Dillion</li>
+      <li><b>Community Organizer</b><br /> Nick Such</li>
+      <li><b>Delivery Lead</b><br /> Todd Willey</li>
+      <li><b>Municipal Partner</b><br /> Erik Schwartz</li>
+    </ul>
+  </div>
+
+  <div class="col-xs-12 col-sm-6 col-lg-4">
+    <h3>Community Partners</h3>
+    <ul>
+      <li><a href="http://www.collexion.net" title="Collexion Hackerspace Website" target="_blank">Collexion</a></li>
+      <li><a href="http://lexladiescode.org" title="Lex Ladies Code Website" target="_blank">LexLadiesCode</a></li>
+      <li><a href="http://maptime.io/lexington" title="Maptime Lexington Website" target="_blank">maptimeLex</a></li>
+      <li><a href="http://runjumpdev.org" title="RunJumpDev Organization Website" target="_blank">RunJumpDev</a></li>
+      <li><a href="http://www.uky.edu/UKHome/" title="University of Kentucky Website" target="_blank">University of Kentucky</a></li>
+    </ul>
+  </div>
+
+  <div class="col-sm-12 col-lg-4">
+    <h3>Municipal Partners</h3>
+    <ul>
+      <li><a href="https://www.lexingtonky.gov" title="Lexington Fayette Urban County Government (LFUCG) Website" target="_blank">Lexington Fayette Urban County Government (LFUCG)</a></li>
+      <li><a href="https://www.lexingtonky.gov/departments/geographic-information-services" 
+      title="Lexington Geographic Information Services Website" target="_blank"> Lexington Geographic Information Services (GIS)</a></li>
```

Comment:
need to add a link to LFUCG SAVI https://www.lexingtonky.gov/substance-abuse-and-violence-intervention